README.md

```diff
@@ -55,7 +55,7 @@ This crate consists of five modules that are provided behind [feature flags](htt
 
 ## Getting Started
 
-This documentation is only for Crossterm version `^0.10`. If you have an older version of Crossterm, then I suggest you check the [Upgrade Manual](https://github.com/TimonPost/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/TimonPost/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
+This documentation is only for Crossterm version `^0.10`. If you have an older version of Crossterm, then I suggest you check the [Upgrade Manual](https://github.com/crossterm-rs/crossterm/blob/master/docs/UPGRADE.md). Also, check out the [examples](https://github.com/crossterm-rs/crossterm/tree/master/examples) folders with detailed examples for all functionality of this crate.
```

Comment:
I'd suggest to remove this whole sentence:

_This documentation is only for Crossterm version `^0.10`_

Just to lower the maintenance burden.